src/spec/doc/_working-with-io.adoc

```diff
@@ -281,7 +281,7 @@ There are also variations of `consumeProcessOutput` that make use of `StringBuff
 `OutputStream` etc... For a complete list, please read the
 http://docs.groovy-lang.org/latest/html/groovy-jdk/java/lang/Process.html[GDK API for java.lang.Process]
 
-In addition, these is a `pipeTo` command (mapped to `|`
+In addition, these are `pipeTo` commands (mapped to `|`
```

Comment:
I think this should be "there is a" or "there's a"

Reply:
You are right, fixed.